src/content/getting-started/architecture/globalnet/_index.en.md

```diff
@@ -97,6 +97,30 @@ replying to DNS queries.
 
 Nothing extra needs to be done to build `submariner-globalnet` as it is built with the standard Submariner build.
 
+## Prerequisites
+
+Allow Globalnet controller to create/update/delete the Service with externalIPs field by below steps:
+
+1. Disable [DenyServiceExternalIPs](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#denyserviceexternalips),
+if enabled
+2. Restrict the use of the Service field externalIPs:
+    * OpenShift: No extra configuration is needed.
+    Default [network.openshift.io/ExternalIPRanger](https://docs.openshift.com/container-platform/4.9/architecture/admission-plug-ins.html)
+    allows the use of Service with externalIPs field only for users that are granted to handle `service/externalips` resource in the
```

Comment:
```suggestion
    allows the use of the `Service` `externalIP` field only for users with permission to handle the `service/externalips` resource in the
```

This is a non-blocker as it's very inconsistent in our docs and the main K8s docs, but I think per the K8s Docs Style Guide `Service` should be `code style` when using it as an API object.

https://kubernetes.io/docs/contribute/style/style-guide/#code-style-inline-code